travis-ci/setup_tests.sh

```diff
@@ -1,23 +1,47 @@
-#!/bin/sh -xe
+#!/bin/bash -xe
 
 # This script starts docker and systemd (if el7)
 
 # Run tests in Container
 # We use `--privileged` for cgroup compatability, which seems to be enabled by default in HTCondor 8.6.x
+
+ENV_FILE=/tmp/travis.env
+for vars in {OSG_RELEASE,OS_VERSION,PKG_SET}; do
+    echo "$vars=${!vars}" >> $ENV_FILE
+done
+
 if [ "${OS_VERSION}" = "6" ]; then
 
-sudo docker run --privileged --rm=true -v /sys/fs/cgroup:/sys/fs/cgroup -v `pwd`:/osg-test:rw centos:centos${OS_VERSION} /bin/bash -c "bash -xe /osg-test/travis-ci/test_inside_docker.sh ${OS_VERSION} ${PACKAGES}"
+    sudo docker run --privileged \
+         --rm=true \
+         --volume=/sys/fs/cgroup:/sys/fs/cgroup \
+         --volume=`pwd`:/osg-test:rw \
+         --env-file=$ENV_FILE \
+         centos:centos${OS_VERSION} \
+         /bin/bash -c "bash -xe /osg-test/travis-ci/test_inside_docker.sh"
 
 elif [ "${OS_VERSION}" = "7" ]; then
 
-docker run --privileged -d -ti -e "container=docker"  -v /sys/fs/cgroup:/sys/fs/cgroup -v `pwd`:/osg-test:rw  centos:centos${OS_VERSION}   /usr/sbin/init
-DOCKER_CONTAINER_ID=$(docker ps | grep centos | awk '{print $1}')
-docker logs $DOCKER_CONTAINER_ID
-docker exec -ti $DOCKER_CONTAINER_ID /bin/bash -xec "bash -xe /osg-test/travis-ci/test_inside_docker.sh ${OS_VERSION} ${PACKAGES};
+    echo "container=docker" >> $ENV_FILE
```

Comment:
where is $container used?

Reply:
/shrug it was in the original `docker run` command